roofit/RDataFrameHelpers/inc/RooAbsDataHelper.h

```diff
@@ -119,6 +119,16 @@ class RooAbsDataHelper : public ROOT::Detail::RDF::RActionImpl<RooAbsDataHelper<
   ///
   /// \param slot When IMT is used, this is a number in the range [0, nSlots) to fill lock free.
   /// \param values x, y, z, ... coordinates of the event.
+
+  // 0 column version
+  void Exec(unsigned int)
+  {
+     if (_eventSize) {
+        throw std::invalid_argument(std::string("RooDataSet can hold ") + std::to_string(_eventSize) +
+                                    " variables per event, but RDataFrame passed 0 columns.");
+     }
+  }
```

Comment:
Why do we need this?  The `if` statement in line 135 seems to be catching also this case.  Also, I guess the documentation above should be associated to the templated version of `Exec()` below.